ha/const.py

```diff
@@ -326,6 +326,7 @@ class ACTUATOR_ATTRIBUTES(ALERT_ATTRIBUTES):
 
 # Health event attribute constants
 class EVENT_ATTRIBUTES:
+    SOURCE = "source"
```

Comment:
This will not be needed in this PR once my PR will be merged

Reply:
I guess, it will be removed when updating the PR after the other PR gets merged.

Reply:
Yes. Right.